Popup/explainer.md

```diff
@@ -49,40 +49,69 @@ We propose a new HTML element called `popup`. This new element can be used for a
 
 `popup` will include:
 
-* A `show()` JavaScript method, for invoking the `popup`. When visible, the `popup` will have a default set of behaviors as well as a default positioning scheme.
+* A few options to invoke/show the `popup`:
+  * A `popup` attribute, applied to whichever element should invoke a given popup (if applicable).
+  * An optional `option` attribute, applied to the `popup` to express that it should be shown.
+  * A `show()` JavaScript method, for invoking the `popup`.
+* When visible, a default set of behaviors as well as a default positioning scheme.
 * Logic for an optional `autofocus` attribute which enables moving focus to the `popup` or to a descendent.
 * An optional `delegatesfocus` attribute, for passing focus to descendants.
 * An optional `anchor` attribute, which both relates the `popup` to an activating element and can be used in a separately-proposed, CSS-based anchor positioning scheme.
-* A `hide()` method for hiding the `popup`
-* [Light dismiss](#light-dismiss) behaviors.
+* A couple means to dismiss the `popup`:
+  * An optional `hidden` attribute, applied to the `popup` to express that it should be hidden.
+  * A `hide()` method for hiding the `popup`.
+  * [Light dismiss](#light-dismiss) behaviors.
 
 ### Showing Popups
 
+#### Option A: the `popup` attribute
+
 One use case for a `popup` is a popup menu triggered by a menu button:
 
 ![A popup menu displayed beneath a button that says "menu". The popup obscures other text on the page.](menu.png)
 
 An author could produce this popup menu using the following markup:
+
 ```html
-<button aria-haspopup="true" aria-controls="menuPopup" id="menuButton">Menu</button>
+<button id="menuButton" popup="menuPopup">Menu</button>
 <popup id="menuPopup" role="menu" anchor="menuButton">
     <!-- Markup for menuitems goes here -->
 </popup>
 ```
 
-Popup menus are not visible until `show()` is called by the author:
+The `popup` attribute on the `button` element (referred to later in this document as the “invoking element”) takes an IDREF pointing to the relevant popup. Invoking this button will likewise invoke the popup with the matching ID.
+
+This `popup` attribute will apply accessibility semantics equivalent to `aria-haspopup="true"` and `aria-controls="menuPopup"` on the button (as well as a “controlled by” reverse relationship mapping on the `popup` itself). This attribute is valid only on a subset of interactive elements:
+
+* `button` or `input` in the button state (`input type="button"`). Invoking one of these elements will show the relevant popup.
+* `input` in the `text`, `email`, `phone`, or `url` states. Setting focus in one of these elements will show the relevant popup. Note: we may need to explore means of suppressing this invocation on focus, for instances where the author instead wishes to show the `popup` based on text-entry logic.
+
+#### Option B: the `open` attribute

```

Comment:
That makes sense, I'll wait for us to close on the `hidden` discussion though because if that is preserved then these operations should effect that attribute too.

Reply:
Fixed! https://github.com/MicrosoftEdge/MSEdgeExplainers/pull/459/commits/23fc0dbd8020f596904c6241f3abe53c4176a742